examples/jsm/loaders/GLTFLoader.js

```diff
@@ -665,7 +665,7 @@ class GLTFMaterialsUnlitExtension {
 
 				const array = metallicRoughness.baseColorFactor;
 
-				materialParams.color.setRGB( ...array, LinearSRGBColorSpace );
+				materialParams.color.setRGB( array[ 0 ], array[ 1 ], array[ 2 ], LinearSRGBColorSpace );
```

Comment:
I think all of the spread operations are technically safe, except this one – the other factors do not allow >3 components. For consistency and safety perhaps it's better that we right them all this way though, thanks!